test/timing_load_creds.c

```diff
@@ -21,6 +21,20 @@
 # include <openssl/bio.h>
 # include "internal/e_os.h"
 
+# ifndef timersub
+/* struct timeval * subtraction; a must be greater than or equal to b */
+#  define timersub(a, b, res)                                         \
+     do {                                                             \
+         (res)->tv_sec = (a)->tv_sec - (b)->tv_sec;                   \
+         if ((a)->tv_usec < (b)->tv_usec) {                           \
+             (res)->tv_usec = (a)->tv_usec + 1000000 - (b)->tv_usec); \
+             --(res)->tv_sec;                                         \
+         } else {                                                     \
+             (res)->tv_usec = (a)->tv_usec - (b)->tv_usec);           \
+         }                                                            \
+     } while(0);
```

Comment:
Drop the trailing `;` which defeats the point of a do-while guard in a macro

Reply:
Done